### ISSUE TITLE
test_identifying_files fails on ciclad

### ISSUE THREAD

**Issue by jypeter** [senesis/climaf#129]
I have run `testing/test_install.sh` at LSCE and on ciclad (in my custom CDAT installation) and `test_identifying_files `is skipped at LSCE, as expected, but fails on ciclad. Is this normal?

### LSCE
`test_identifying_files (test_1.B_CMIP5_DRS_Ciclad) ... skipped 'because not on Ciclad'`

### ciclad
```
test_identifying_files (test_1.B_CMIP5_DRS_Ciclad) ... error    : "This dataset is ambiguous on attribute version='*'; please choose among : ['latest', 'v20110701'] or use either 'ensure_dataset=False' (with method baseFiles or listfiles) or 'option='choices' (with method explore)"
ERROR

======================================================================
ERROR: test_identifying_files (test_1.B_CMIP5_DRS_Ciclad)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "test_1.py", line 109, in test_identifying_files
    files=self.ds.baseFiles()
  File "/data/jypmce/cdat/climaf/climaf/classes.py", line 677, in baseFiles
    if ensure_dataset : self.explore()
  File "/data/jypmce/cdat/climaf/climaf/classes.py", line 664, in explore
    (kw,dic[kw],entry))
Climaf_Classes_Error: "This dataset is ambiguous on attribute version='*'; please choose among : ['latest', 'v20110701'] or use either 'ensure_dataset=False' (with method baseFiles or listfiles) or 'option='choices' (with method explore)"

----------------------------------------------------------------------
Ran 7 tests in 9.139s

FAILED (errors=1, skipped=2)
```

**Comment by senesis** [comment]
We have moved the handling of CMIP5 data from a specific function to a 'generic' one, and CliMAF then can no more understand that version=* should be interpreted as version=latest in CMIP5 (nor in CMIP6). We must then change the test (but this not the most important impact of the change ...). Will be done for next version. 
It wiould help if you could test changing line 106 of test_1.py to :
`        self.ds=ds(experiment="1pctCO2", variable="tas", period="1860-1861",version="latest")
`

**Comment by jypeter** [comment]
It works fine after changing line 106!
```
test_selecting_files (test_1.B_CMIP5_DRS_CNRM) ... skipped "because CNRM's Lustre not available"
test_identifying_files (test_1.B_CMIP5_DRS_Ciclad) ... ok
test_selecting_files (test_1.B_CMIP5_DRS_Ciclad) ... ok
test_selecting_CACO3_for_IPSL_CM4 (test_1.C_OCMIP5_CIclad) ... ok

----------------------------------------------------------------------
Ran 9 tests in 13.306s
```

**Comment by senesis** [comment]
FIxed for next version (1.2.11)
